docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
+  database:
+    container_name: DarkflameDatabase
+    image: mariadb:latest
```

Comment:
I don't think using `latest` is a great idea. Inevitably leads to brokenness over time when the image gets updated with a breaking change. Maybe lock it to the current version.

Reply:
Yes, this is true. Will change it to a tagged version.